packages/agent/src/dwn-manager.ts

```diff
@@ -184,8 +188,11 @@ export class DwnManager {
       });
       dwnRpcRequest.message = message;
       messageData = data;
-
+    } else if ('message' in request) {
```

Comment:
@diehuxx The approach to sending a `PermissionsGrant` is different than how both Record and ProtocolsConfigure messages are sent to a remote DWN. What was the motivation for introducing a new message property to the send request object?

Reply:
I've been wanting to make that addition separate of `PermissionsGrants`. The existing code forces an existing message to be turned into `options` then re built and signed. Why not just send the existing message?